docs/redux/redux-intro.md

```diff
@@ -19,14 +20,92 @@ We recommend using the most recent version of Redux. To see which version of Red
 
 The entire **[state](https://redux.js.org/understanding/thinking-in-redux/glossary#state)** of the application will be represented by one JavaScript object, a **[store](https://redux.js.org/understanding/thinking-in-redux/glossary#store)**.
 
+This makes it easy to create universal apps, as the state from your server can be serialized and hydrated into the client with no extra coding effort. A single state tree also makes it easier to debug or inspect an application; it also enables you to persist your app's state in development, for a faster development cycle. Some functionality which has been traditionally difficult to implement - Undo/Redo, for example - can suddenly become trivial to implement, if all of your state is stored in a single tree.
```

Comment:
I think the previous version of the explanation is enough for the `Three Principles of Redux`.
If you want to explain more, how about adding below at the end?
```
Please find more information [here](https://redux.js.org/understanding/thinking-in-redux/three-principles).